docs/securing-webhooks.md

```diff
@@ -0,0 +1,36 @@
+# Securing webhooks
```

Comment:
I'd like to start putting together some scrappy docs about managing the operator in production. Over time we can make them better and add them to our main docs site. But for now, I figured it was a good idea to just get some things written down.